pkg/controllers/dashboardapi/settings/settings.go

```diff
@@ -91,6 +91,7 @@ func (s *settingsProvider) SetAll(settingsMap map[string]settings.Setting) error
 				Default: setting.Default,
 			}
 			if value != "" {
+				newSetting.Source = "env"
 				newSetting.Value = value
```

Comment:
Unrelated to your PR, but can we change the `value` variable name to `envValue`. As it is only ever used in that manner.